int32 ID_node
string type
string node_name

float32 utiliz_tot
float32[] utiliz_array
float32[] queue_array
time[] time_array
string info
int32 queue_left

int32 events_left

bool stop_esecution
bool statistic